src/assets/js/archive.js

```diff
@@ -161,21 +208,27 @@ function updateDownloadLinkFailed(os) {
 }
 
 function getProvenanceLink(os, release, date, channel) {
-  const baseUrl = 'https://storage.googleapis.com/flutter_infra_release/releases/'
+  const baseUrl = 'https://storage.googleapis.com/flutter_infra_release/releases/';
   if (os === 'windows' && date < new Date(Date.parse('4/3/2023'))) {
     // provenance not available before 4/3/2023 for Windows
-    return $('<span />').text('-');
+    const spanElement = document.createElement('span');
+    spanElement.textContent = '-';
+    return spanElement;
   } else if (date < new Date(Date.parse('12/15/2022'))) {
     // provenance not available before 12/15/2022 for macOS and Linux
-    return $('<span />').text('-');
+    const spanElement = document.createElement('span');
+    spanElement.textContent = '-';
+    return spanElement;
   }
+
   const extension = os === 'linux' ? 'tar.xz' : 'zip';
-  return $('<a />').attr('href',
-    `${baseUrl}${channel}/${os}/flutter_${os}_${release.version}-${channel}`+
-    `.${extension}.intoto.jsonl`
-  ).text(`${release.version} file`)
+  const aElement = document.createElement('a');
```

Comment:
Consider renaming this variable to `provenanceAnchor` or similar for a more descriptive name.

Reply:
I solved this on this commit #71881ab, using your name suggetion. Thanks.